src/datatransfer.js

```diff
@@ -19,6 +19,14 @@ export default class DataTransfer {
 		 * @member {DataTransfer} #_native
 		 */
 		this._native = nativeDataTransfer;
+
+		/**
+		 * The array of files created from the native `DataTransfer#files` and `DataTransfer#items` objects.
```

Comment:
This is confusing – it indicates that `#files` is a combination of these two native properties while it's either one or another.